arrebol/worker/auth/token/token.go

```diff
@@ -0,0 +1,11 @@
+package token
+
+type Token interface {
```

Comment:
do you guys plan more than one token implementation?

Reply:
No. So I removed that interface.